### PR TITLE
Send link data attributes in link:external event

### PR DESCRIPTION
**Pull request by esseb**

### CODE REVIEW

File: src/lib/history.js [hunk 1]
```diff
@@ -157,7 +159,7 @@ class History {
       req.reset();
       res.reset();
       // Set flag for use downstream
-      req.cached = (res.cached = true);
+      req.cached = res.cached = true;
```
Comment (popeindustries) [inline src/lib/history.js]: run prettier?

File: src/lib/history.js [hunk 2]
```diff
@@ -231,7 +233,16 @@ class History {
 
     // Cross origin
     if (!sameOrigin(el.href)) {
-      return void this.fn(el.href);
+      const data = {};
+
+      const attributes = Array.prototype.slice.call(el.attributes);
+      attributes.forEach(attribute => {
```
Comment (popeindustries) [inline src/lib/history.js]: missing new line between

File: src/lib/history.js (hunk 1 above)
Comment (esseb) [reply src/lib/history.js]: Prettier v1.7.4 removed the parentheses.

File: src/lib/history.js (hunk 2 above)
Comment (esseb) [reply src/lib/history.js]: Fixed

**Pull request merged**